mbed_devices/mbed_tools/cli.py

```diff
@@ -22,15 +44,26 @@ def _build_output(devices: Iterable[Device]) -> str:
     return tabulate(devices_data, headers=headers)
 
 
-@click.command()
-def list_connected_devices():
-    """Prints connected devices."""
-    devices = get_connected_devices()
-    if devices:
-        output = _build_output(devices)
-        click.echo(output)
-    else:
-        click.echo("No connected Mbed devices found.")
-
-
-cli = list_connected_devices
+def _build_json_output(devices: Iterable[Device]) -> str:
+    devices_data = []
+    for device in devices:
+        mbed_target = device.mbed_target
+        if mbed_target:
+            mbed_target_data = {
```

Comment:
So we want to only include the target data in the json output but not the table?

Reply:
Couple of reasons actually:
- replicates `mbed-ls`, so imo easier transition for existing users
- tabular output containing all the info about target would be extremely long and mess up the table

This is flexible enough so that we can adjust the output quite easily at later stage if need be. :) 
I was actually wondering if I should add `product_code` to the tabular output, but I don't want to make those decisions rn.

Reply:
Ok, thats fair enough 👍